searchobject/src/main/java/org/jalau/at18/searchobject/controller/endpoint/FaceDetectionController.java

```diff
@@ -0,0 +1,64 @@
+package org.jalau.at18.searchobject.controller.endpoint;
+
+/**
+ * Copyright (c) 2022 Jala University.
+ *
+ * This software is the confidential and proprieraty information of Jalasoft
```

Comment:
Typo in `proprieraty`, maybe you meant `property`?

Reply:
already change it